bears/bears_router.js

```diff
@@ -0,0 +1,95 @@
+// bring in express 
+const express = require('express');
+
+// create express router
+const router = express.Router();
+
+const BearsModel = require('./bears_model');
+
+
+// GET all bears
+router.get('/', async (req, res) => {
+    try {
+        const br = await BearsModel.find()
```

Comment:
I'd be more discriptive here. The Good thing about Async and Await is how declaritive it is. It's better to make the naming logical and to the point here.